docs/sync.js

```diff
@@ -50,6 +51,14 @@
  *    file to complete and then open. If not set, the Realm will be downloaded before opened.
  */
 
+ /**
+ * This describes the different options used to create a {@link Realm} instance with Realm Platform synchronization.
```

Comment:
"Realm Platform" - that's not a term we use. Also in line 38, it's called "Realm Cloud", which is also not a term we use :-)
It's "MongoDB Realm"

Reply:
Damn, just copied from v6.x. I'll fix it - thanks!